src/main/java/org/gridsuite/securityanalysis/server/service/SecurityAnalysisWorkerService.java

```diff
@@ -125,26 +145,75 @@ private static String sanitizeParam(String param) {
         return Mono.zip(network, contingencies)
                 .flatMap(tuple -> {
                     SecurityAnalysis securityAnalysis = configService.getSecurityAnalysisFactory().create(tuple.getT1(), LocalComputationManager.getDefault(), 0);
-                    CompletableFuture<SecurityAnalysisResult> result = securityAnalysis.run(VariantManagerConstants.INITIAL_VARIANT_ID, context.getParameters(), n -> tuple.getT2());
-                    return Mono.fromCompletionStage(result);
+                    CompletableFuture<SecurityAnalysisResult> future = securityAnalysis.run(VariantManagerConstants.INITIAL_VARIANT_ID, context.getParameters(), n -> tuple.getT2());
+                    if (resultUuid != null) {
+                        futures.put(resultUuid, future);
+                    }
+                    if (resultUuid != null && cancelComputationRequests.get(resultUuid) != null) {
+                        return Mono.empty();
+                    } else {
+                        return Mono.fromCompletionStage(future);
+                    }
                 });
     }
 
     @Bean
-    public Consumer<Flux<Message<String>>> consumeRun() {
-        return f -> f.log(CATEGORY_BROKER_INPUT, Level.FINE)
```

Comment:
why remove the log for each message ? it's useful for debugging

Reply:
Done